<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDPayWindow</class>
 <widget class="QMainWindow" name="WBSDPayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Payment/RFID settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>1091</width>
      <height>566</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblRFIDTimeOut">
         <property name="text">
          <string>RFID Timeout x 10 sec</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cboClearCreditTime"/>
       </item>
       <item row="3" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblPayEnabled">
         <property name="text">
          <string>Payment enabled</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblRFIDMode">
         <property name="text">
          <string>RFID Mode</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblCurrency">
         <property name="text">
          <string>Currency</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cboSaleType"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblSaletype">
         <property name="text">
          <string>Vendor mode</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblClearCreditTime">
         <property name="text">
          <string>Clear credit time (minutes)</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="cboCurrency">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cboRFIDMode"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cboRFIDTimeout"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cboPaymentEnabled"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="button9">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button10">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button11">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0" columnminimumwidth="0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="horizontalSpacing">
        <number>50</number>
       </property>
       <property name="verticalSpacing">
        <number>9</number>
       </property>
       <item row="2" column="2">
        <widget class="QPushButton" name="button6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Price Button 6</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="button3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Price Button 3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Price Button 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="button4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Price Button 4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="button1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Price Button 1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="button5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Price Button 5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="button7">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Price Button 7</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Price Button 8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="horizontalSpacing">
        <number>32</number>
       </property>
       <property name="verticalSpacing">
        <number>9</number>
       </property>
       <item row="0" column="5">
        <widget class="QPushButton" name="btnCoinChanelF">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Coin Channel F</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnCoinChanelA">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Coin Channel A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btnCoinChanelE">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Coin Channel E</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnCoinChanelB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Coin Channel B</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnCoinChanelD">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Coin Channel D</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnCoinChanelC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Coin Channel C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_cashbox" columnstretch="1,3,1,1">
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_setfull_2">
         <property name="text">
          <string>Set Value</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Cashbox Full Flag</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btn_setreset_2">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Cashbox Warning Set Value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Cashbox Full Set Value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Cashbox Coin Counters</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btn_setwarn_2">
         <property name="text">
          <string>Set Value</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBox_3">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_4">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Cashbox Warning Flag</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lbl_flag_full_2">
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lbl_flag_warn_2">
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lbl_flag_counters">
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="6">
        <widget class="QPushButton" name="btnReset">
         <property name="text">
          <string>Collect cashbox</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="lblMoneyInCashBox">
         <property name="text">
          <string>Money in cashbox</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblFreeVendTokens">
         <property name="text">
          <string>Free vend tokens</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblTotalCollected">
         <property name="text">
          <string>Total collected money</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblTotalTurnOver">
         <property name="text">
          <string>Total turnover</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cboRFIDMode</tabstop>
  <tabstop>cboRFIDTimeout</tabstop>
  <tabstop>cboPaymentEnabled</tabstop>
  <tabstop>cboClearCreditTime</tabstop>
  <tabstop>cboSaleType</tabstop>
  <tabstop>cboCurrency</tabstop>
  <tabstop>button9</tabstop>
  <tabstop>button10</tabstop>
  <tabstop>button11</tabstop>
  <tabstop>button7</tabstop>
  <tabstop>button1</tabstop>
  <tabstop>button4</tabstop>
  <tabstop>button8</tabstop>
  <tabstop>button2</tabstop>
  <tabstop>button5</tabstop>
  <tabstop>button3</tabstop>
  <tabstop>button6</tabstop>
  <tabstop>btnCoinChanelA</tabstop>
  <tabstop>btnCoinChanelB</tabstop>
  <tabstop>btnCoinChanelC</tabstop>
  <tabstop>btnCoinChanelD</tabstop>
  <tabstop>btnCoinChanelE</tabstop>
  <tabstop>btnCoinChanelF</tabstop>
  <tabstop>btnReset</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>763</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
